zcash_client_backend/src/data_api.rs

```diff
@@ -662,12 +662,22 @@ impl<NoteRef> SpendableNotes<NoteRef> {
         self.sapling.as_ref()
     }
 
+    /// Consumes this value and returns the Sapling notes contained within it.
+    pub fn take_sapling(self) -> Vec<ReceivedNote<NoteRef, sapling::Note>> {
```

Comment:
It is used in conjunction with the fix to #1260 inside the implementation of `ShieldedPoolTester` for Sapling (and similarly for Orchard). So it could instead be made crate-private now that `SaplingPoolTester` is also in `zcash_client_backend` instead of documenting it as an addition.